src/index.scss

```diff
@@ -1,4 +1,34 @@
 :root {
+  --screen-mobile-s: 320px;
+  --screen-mobile-m: 375px;
+  --screen-mobile-l: 425px;
+  --screen-tablet: 768px;
+  --screen-laptop: 1024px;
+  --laptop-l: 1440px;
+  --laptop-4k: 2560px;
+
+  --common-font: 'SST W20 Roman', sans-serif;
+  --common-font-bold: 'SST W20 Bold', sans-serif;
+  --common-font-en: 'SST W20 Roman', sans-serif;
+  --common-font-en-bold: 'SST W20 Bold', sans-serif;
```

Comment:
use only en fonts, remove W55.  names should be: SSTW20-Bold, SSTW20-Roman. check this pr and font naming: https://github.com/weblodz/folio/pull/3